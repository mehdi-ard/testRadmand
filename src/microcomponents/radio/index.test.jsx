import React from 'react';
import renderer from 'react-test-renderer';
import Radio from '.';

it('renders defult', () => {
  expect(renderer.create(<Radio>test</Radio>).toJSON()).toMatchSnapshot();
});

it('renders default checked', () => {
  expect(
    renderer.create(<Radio checked>test</Radio>).toJSON()
  ).toMatchSnapshot();
});

it('renders disabled', () => {
  expect(
    renderer.create(<Radio disabled={true}>disabled</Radio>).toJSON()
  ).toMatchSnapshot();
});
