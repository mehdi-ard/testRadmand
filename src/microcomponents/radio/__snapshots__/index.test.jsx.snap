// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders default checked 1`] = `
<label
  className="ant-radio-wrapper ant-radio-wrapper-checked"
>
  <span
    className="ant-radio ant-radio-checked"
    style={Object {}}
  >
    <input
      checked={true}
      className="ant-radio-input"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      type="radio"
    />
    <span
      className="ant-radio-inner"
    />
  </span>
  <span>
    test
  </span>
</label>
`;

exports[`renders defult 1`] = `
<label
  className="ant-radio-wrapper"
>
  <span
    className="ant-radio"
    style={Object {}}
  >
    <input
      checked={false}
      className="ant-radio-input"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      type="radio"
    />
    <span
      className="ant-radio-inner"
    />
  </span>
  <span>
    test
  </span>
</label>
`;

exports[`renders disabled 1`] = `
<label
  className="ant-radio-wrapper ant-radio-wrapper-disabled"
>
  <span
    className="ant-radio ant-radio-disabled"
    style={Object {}}
  >
    <input
      checked={false}
      className="ant-radio-input"
      disabled={true}
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      type="radio"
    />
    <span
      className="ant-radio-inner"
    />
  </span>
  <span>
    disabled
  </span>
</label>
`;
